content/en/docs/concepts/storage/volumes.md

```diff
@@ -1217,7 +1217,7 @@ persistent volume:
 
 #### CSI raw block volume support
 
-{{< feature-state for_k8s_version="v1.11" state="alpha" >}}
```

Comment:
feature-gates.md should also be updated

Reply:
The section below on the feature being turned off by default needs to be updated

Reply:
Done